examples/rustfmt/module/rust/src/proto/rustfmt_grpc.rs

```diff
@@ -29,22 +29,14 @@ pub trait FormatServiceNode {
 }
 
 // Oak Node gRPC method dispatcher
-pub fn dispatch(node: &mut dyn FormatServiceNode, method: &str, req: &[u8], out_handle: oak::WriteHandle) {
-    let mut out = oak::io::Channel::new(out_handle);
+pub fn dispatch(node: &mut dyn FormatServiceNode, method: &str, req: &[u8], mut writer: grpc::ChannelResponseWriter) {
```

Comment:
Is `mut` actually needed? I'm slightly confused by it, what error do you get if you omit it?

Reply:
Ah I see; given all the other params are references already, it may make sense to have `writer: &mut grpc::ChannelResponseWriter` instead? I have not seen mut owned parameters used very much. Up to you anyways.

Reply:
I have an example application in preparation which saves off the `ChannelResponseWriter` object, so moving ownership is (much) easier for that&hellip;